package.json

```diff
@@ -6,7 +6,7 @@
   },
   "scripts": {
     "type-check": "tsc --noEmit",
-    "commitmsg": "commitlint -e $GIT_PARAMS",
+    "commitmsg": "commitlint -q -e $GIT_PARAMS",
```

Comment:
Why is this needed?

Reply:
I kept getting a `formatted.join` when executing `commitlint` commands. Absolutely no idea why and I didn't feel like investigating the library.

Reply:
But I agree that I shouldn't quiet it. It totally disables the error output which makes fixinx lint errors annoying.

Reply:
Regarding this, I also noticed a deprecation warning:
```
$ yarn run commitmsg
yarn run v1.19.0
$ commitlint -e $GIT_PARAMS
Using environment variable syntax ($GIT_PARAMS) in -e |--edit is deprecated. Use '{-E|--env} HUSKY_GIT_PARAMS instead'
C:\Users\Telokis\Documents\git\verdaccio-gitlab\node_modules\@commitlint\cli\lib\cli.js:119
        throw err;
        ^

Error: Received $GIT_PARAMS as value for -e | --edit, but GIT_PARAMS or HUSKY_GIT_PARAMS are not available globally.
    at getEditValue (C:\Users\Telokis\Documents\git\verdaccio-gitlab\node_modules\@commitlint\cli\lib\cli.js:307:9)
    at normalizeFlags (C:\Users\Telokis\Documents\git\verdaccio-gitlab\node_modules\@commitlint\cli\lib\cli.js:268:15)
    at C:\Users\Telokis\Documents\git\verdaccio-gitlab\node_modules\@commitlint\cli\lib\cli.js:126:11
    at new Promise (<anonymous>)
    at main (C:\Users\Telokis\Documents\git\verdaccio-gitlab\node_modules\@commitlint\cli\lib\cli.js:123:9)
    at Object.<anonymous> (C:\Users\Telokis\Documents\git\verdaccio-gitlab\node_modules\@commitlint\cli\lib\cli.js:115:1)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I can easily fix it by removing `$GIT_PARAMS` from the arguments.

Reply:
I pushed this fix and reverted the `-q`